nvflare/fuel/flare_api/flare_api.py

```diff
@@ -437,19 +437,24 @@ def basic_cb_with_print(session: Session, job_id: str, job_meta, *cb_args, **cb_
     return True
 
 
-def new_secure_session(username: str, startup_kit_location: str, debug: bool = False, timeout: float = 10.0) -> Session:
+def new_secure_session(username: str,
+                       startup_kit_location: str,
+                       debug: bool = False,
+                       timeout: float = 10.0,
+                       secure_mode: bool = True) -> Session:
```

Comment:
Why do we need secure_mode here? This method is intended for creating a secure session.
If you want insecure session, use new_insecure_session() method.